### ISSUE TITLE
FP in 'AD Admin Not Configured For SQL Server' query for Terraform

### ISSUE THREAD

**Issue by t3mi** [Checkmarx/kics#3437]
### Platform
*(e.g. Terraform, Ansible, etc.)*
Terraform
### Provider
*(e.g. AWS, Azure, GCP, etc.)*
Azure
### Description
Following code produces false positive result.
```hcl
resource "azurerm_sql_server" "test" {
  name                         = "test-sql"
  resource_group_name          = "rgname"
  location                     = "eastus2"
  version                      = "12.0"
  administrator_login          = "admin"
  administrator_login_password = "password"
}

resource "azurerm_sql_active_directory_administrator" "test" {
  server_name         = azurerm_sql_server.test.name
  resource_group_name = azurerm_resource_group.test.name
  login               = "testlogin"
  tenant_id           = "ee591259-1c6a-4108-a3ed-7726f6c925d9"
  object_id           = "98cb95ec-38a2-4f92-ae03-6aef806dc481"
}

```
